app/templates/environmental-solutions.hbs

```diff
@@ -0,0 +1,21 @@
+<div class="header">
+    <h1>Environmental Solutions</h1>
+</div>
+<div class="content">
+    <form class="pure-form search" {{action "search" on="submit"}}>
+        <fieldset>
+            <div class="pure-g">
+                <div class="pure-u-1">
+                    <label>Keyword</label>
+                  {{input class="pure-input-1" value=qField}}
```

Comment:
About `value=qField`, eccn implementation as a guide, I can see the definition is done here https://github.com/GovWizely/explorer/blob/master/app/controllers/eccns.js#L7.
I suppose in my case it should be exactly the same correct?